packages/starlight/utils/routing.ts

```diff
@@ -35,7 +35,7 @@ interface Path extends GetStaticPathsItem {
 const normalizeIndexSlug = (slug: string) => (slug === 'index' ? '' : slug);
 
 /** All entries in the docs content collection. */
-const docs: StarlightDocsEntry[] = (await getCollection('docs')).map(({ slug, ...entry }) => ({
+const docs: StarlightDocsEntry[] = (await getCollection('docs'))?.map(({ slug, ...entry }) => ({
```

Comment:
Thanks for reporting this and following up @TheOtterlord!

Does this really stop things crashing? I’d think that the optional chaining here would result in the `docs` variable being `undefined` and then we’d get similar problems downstream with places that use `docs`?

Reply:
wait, how'd I mess a single line up :sweat_smile: 
this should be a nullcheck to make it an array. I'll update this now